### ISSUE TITLE
CS8604 reporting false positive for XAttribute when using nullables

### ISSUE THREAD

**Issue by DrGriff** [dotnet/roslyn#48005]
**Version Used**: 
FxCop 3.3.0
StyleCop.Analyzers 1.1.118
.Net 5 (5.0.100-rc.1.20452.10)

**Steps to Reproduce**:

I'm using .Net Core 5 (still in Preview) and I have nullables enabled.
```
    <TargetFramework>net5.0</TargetFramework>
    <Nullable>enable</Nullable>
```

When dealing with an `XElement` then I'm finding a warning for the code:
```
myXElement.Attribute("name")?.Value .......
```
I'm seeing a Warning (Error in my case as I treat all Warnings as Errors):

> CS8604: Possible null reference for parameter 'name' in 'XAttribute? XElement.Attribute(XName name)'.
> 
> Possible null reference argument for parameter 'name' in 'System.Xml.Linq.XElement.Attribute'.

**Expected Behavior**:
When passing a literal value (in my case "name") that would create a valid XName, then I would not expect to see a warning.

If of course the literal value was invalid in some form then I'd expect to get an error, but not this one.

**Actual Behavior**:
A false-positive warning of a possible null exception.

**Comment by RikkiGibson** [comment]
Thanks for reporting this bug. This has been fixed in #47555. The fix should be available in the next .NET 5 RC.

**Comment by sharwell** [comment]
Duplicate of #39802
